server/src/model/scd.ts

```diff
@@ -0,0 +1,191 @@
+import * as D from 'io-ts/Decoder';
+import { fold } from 'fp-ts/lib/Either';
+import { pipe } from 'fp-ts/lib/function';
+/* --------------------------------------------------------------------------------------------
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for license information.
+ * ------------------------------------------------------------------------------------------ */
```

Comment:
Please put the copyright information on the first line:

```suggestion
/* --------------------------------------------------------------------------------------------
 * Copyright (c) Microsoft Corporation. All rights reserved.
 * Licensed under the MIT License. See License.txt in the project root for license information.
 * ------------------------------------------------------------------------------------------ */

import * as D from 'io-ts/Decoder';
import { fold } from 'fp-ts/lib/Either';
import { pipe } from 'fp-ts/lib/function';
```